Common/ItemDropping/DropTable.cs

```diff
@@ -0,0 +1,80 @@
+﻿using PathOfTerraria.Content.Items.Consumables.Maps;
+using PathOfTerraria.Content.Items.Currency;
+using PathOfTerraria.Content.Items.Gear;
+using PathOfTerraria.Core.Items;
+using System.Collections.Generic;
+using System.Linq;
+using Terraria.Utilities;
+
+namespace PathOfTerraria.Common.ItemDropping;
+
+internal class DropTable
+{
+	/// <summary>
+	/// Drops an item based off of the following table:<br/>
+	/// <c>Gear:</c> Default 8%<br/>
+	/// <c>Currency:</c> Default 15%<br/>
+	/// <c>Maps:</c> Default 5%<br/>
+	/// </summary>
+	/// <param name="random">The random to 'seed' the choice with. Defaults to Main.rand.</param>
+	public static ItemDatabase.ItemRecord RollMobDrops(int itemLevel, float dropRarityModifier, float gearChance = 0.8f, float currencyChance = 0.15f, float mapChance = 0.05f, 
```

Comment:
Is there a reason these were changed from 
90% Gear
7% Currency
3% Map?

Reply:
80-15-5 is the intended future rate, no?

Reply:
Ahh, I see where that came from. The numbers from the project ticket were a bit of a example, not real rates. That said I will probably adjust them now that I can play with the table so we can leave as it is